content/docs/2.10/deploy.md

```diff
@@ -106,13 +116,19 @@ VERSION=2.10.0 make deploy
 - In case of installing from released YAML file just run the following command (if needed, replace the version, in this case `2.10.0`, with the one you are using):
 
 ```sh
```

Comment:
Comments in shell are using # instead of // which breaks the syntax highlighting